extension/src/prototypes/view/selection/ColorSchemeContent.tsx

```diff
@@ -69,9 +69,19 @@ const QualitativeContent: FC<{
   continuous?: boolean;
   onClose?: () => void;
 }> = ({ colorScheme, continuous = false, onClose }) => {
-  const colors = useAtomValue(
-    useMemo(() => atom(get => get(colorScheme.colorsAtom)), [colorScheme]),
-  );
+  const [colors, setColors] = useAtom(colorScheme.colorsAtom);
+  const [originalColors, setOriginalColors] = useState(colors);
+
+  useEffect(() => {
+    if (!originalColors) {
+      setOriginalColors(colors);
+    }
+  }, [originalColors, colors]);
```

Comment:
Is this in use?

Reply:
ColorScheme is expected to have a value, so should never be undefined. In this case, this useEffect won't be needed. I added it here just because if you add a prop value to a useState on initiate, but the value is undefined, that state won't be updated if the prop finally gets a value and the component rerenders.
This is probably overkill so can remove it, but thought it was safer to leave just in case the prop type was wrong..

Reply:
Emmm i mean is `originalColors` in use? I didn't find it been used anywhere except in the useEffect. When handle reset you're using `defaultColors` not `originalColors`.

Reply:
Removed!